### PR TITLE
looking-glass-client: B1 -> B2

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change
Update looking glass.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/virtualization/looking-glass-client/default.nix [hunk 1]
```diff
@@ -41,9 +39,9 @@ stdenv.mkDerivation rec {
       step required to move away from dual booting with other operating systems
       for legacy programs that require high performance graphics.
     '';
-    homepage = "https://looking-glass.hostfission.com/";
+    homepage = https://looking-glass.hostfission.com/;
```
Comment (omasanori) [inline pkgs/applications/virtualization/looking-glass-client/default.nix:42]:
```suggestion
    homepage = "https://looking-glass.io/";
```
The website seems to be moved. Also, the unquoted URL is discouraged as per [RFC 45](https://github.com/NixOS/rfcs/blob/master/rfcs/0045-deprecate-url-syntax.md).

Comment (Chiiruno) [reply pkgs/applications/virtualization/looking-glass-client/default.nix]: Done.

**Review by alexbakker — APPROVED** [review]
LGTM

**Comment by Chiiruno** [comment]
cc @Mic92 
Not sure how things are done these days, but could we get this merged please?

File: pkgs/applications/virtualization/looking-glass-client/default.nix [hunk 2]
```diff
@@ -1,35 +1,33 @@
...
-
   sourceRoot = "source/client";
 
-  installPhase = ''
-    mkdir -p $out/bin
-    mv looking-glass-client $out/bin
+  # Fix some sort of AVX compiler problem.
+  preConfigure = ''
+    sed -i "s/-ggdb>\"/-ggdb>\"\n  \"-mavx\"/" CMakeLists.txt
   '';
```
Comment (Mic92) [inline pkgs/applications/virtualization/looking-glass-client/default.nix:31]: Does this work?

```suggestion
  # Fix some sort of AVX compiler problem.
  NIX_CFLAGS_COMPILE = "-mavx";
```

Comment (Chiiruno) [reply pkgs/applications/virtualization/looking-glass-client/default.nix]: Yes, thank you for the much more elegant fix!

**Pull request merged**